src/main/webapp/partials/instructorHelpProfile.jsp

```diff
@@ -0,0 +1,209 @@
+                <br><br>
+                <div class="well well-plain">
+                    <button type="button" class="btn btn-default btn-xs icon-button pull-right"
+                            id="button_add_comment" data-toggle="tooltip"
+                            data-placement="top" title="Add comment">
+                        <span class="glyphicon glyphicon-comment glyphicon-primary"></span>
+                    </button>
+                    <div class="form form-horizontal" id="studentInfomationTable">
+                        <div class="form-group">
+                            <label class="col-sm-1 control-label">Student Name:</label>
+                            <div class="col-sm-11">
+                                <p class="form-control-static">Alice Betsy</p>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label class="col-sm-1 control-label">Section Name:</label>
+                            <div class="col-sm-11">
+                                <p class="form-control-static">Section A</p>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label class="col-sm-1 control-label">Team Name:</label>
+                            <div class="col-sm-11">
+                                <p class="form-control-static">Team A</p>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label class="col-sm-1 control-label">Official Email Address:</label>
+                            <div class="col-sm-11">
+                                <p class="form-control-static">alice@email.com</p>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label class="col-sm-1 control-label">Comments:</label>
+                            <div class="col-sm-11">
+                                <p class="form-control-static">Alice is a transfer student.</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                The <b>Section Name</b> will appear only if the sections were created for the respective course.
+                <br><b>Team Name</b> is the name of the team to which the student belongs to, or displays <b>'NA'</b> if the student does not belong to a team.
+                <br><b>Official Email Address</b> is the email address used to contact the student.
+                <br><b>Comments</b> is any information you mentioned about the student during enrollment.
+                <br>Below this is the <b>'Mode Info'</b> section where you can see the personal description given by the student. It will look something like this -
```

Comment:
This should be on a separate paragraph.

Below this is the 'Mo~~d~~**r**e Info' section where you can see ~~the~~ **a** personal description given by the student **, if any**. It will look something like this: